README.md

```diff
@@ -59,16 +59,39 @@ Here is a list of goals in decreasing order of importance:
 * [i3](https://github.com/i3/i3)
 * [RapidJSON](https://github.com/Tencent/rapidjson/)
 * Modern C++17 compiler.
+* CMake (optional)
 
 ## Installation ##
 
+### Install via `make`
+
 ```sh
 ./configure --help    # See how you can configure build system.
 ./configure [options] # Configure build for your needs.
 make                  # Build library in configured way.
 # make install        # Optionally install the library in configured way.
 ```
 
+### Install via `cmake`
```

Comment:
Please distinguish official Make build from optional CMake build for now, since I don't feel confident about reviewing/maintaining it at the moment.
```suggestion
### Install via `cmake` (unofficial - Thanks @rbrugo)
```
Of course, feel free to leave your username as text, link, or remove it altogether, whichever you prefer.